transmart-core-api/src/main/groovy/org/transmartproject/core/querytool/QueriesResource.groovy

```diff
@@ -33,6 +33,17 @@ interface QueriesResource {
     QueryResult runQuery(QueryDefinition definition, String username) throws InvalidRequestException
 
     /**
+     * Creates and executes a query in one go to update queries resource data. The query is run synchronously.
+     *
+     * @param definition the definition to use
+     * @param username the user that issued the query. This is the username of
+     * a tranSMART user (for usages in tranSMART) or an i2b2 user, for
+     * compatibility with i2b2.
+     * @return the resulting query result
+     */
+    QueryResult runDisablingQuery(Long id, String username) throws InvalidRequestException
```

Comment:
This is the function to disable a query, right? Suggestion: name it `disableQuery` and return `void`.